service-springboot-rest/README.md

```diff
@@ -100,5 +100,5 @@ Integration test of the Quickstart
 1. Make sure you have an <span>Keycloak</span> server running with an admin user in the `master` realm or use the provided docker image
 2. Be sure to set the `TestHelper.keycloakBaseUrl` in the `createArchive` method (default URL is localhost:8180/auth).
 3. Set accordingly the correct url for the `keycloak.auth-server-url` in the test [application.properties](src/test/resources/application.properties).
-4. Run `mvn test`
+4. Run `mvn test -Pspring-boot`
```

Comment:
@martin-kanis did you get the following output while running tests?
```
Tests run: 2, Failures: 0, Errors: 2, Skipped: 0, Time elapsed: 1.43 sec <<< FAILURE! - in org.keycloak.quickstart.springboot.ProductServiceTest
org.keycloak.quickstart.springboot.ProductServiceTest  Time elapsed: 1.429 sec  <<< ERROR!
javax.ws.rs.BadRequestException: HTTP 400 Bad Request
        at org.keycloak.quickstart.springboot.ProductServiceTest.setup(ProductServiceTest.java:32)

org.keycloak.quickstart.springboot.ProductServiceTest  Time elapsed: 1.429 sec  <<< ERROR!
javax.ws.rs.NotFoundException: HTTP 404 Not Found
        at org.keycloak.quickstart.springboot.ProductServiceTest.cleanUp(ProductServiceTest.java:60)


Results :

Tests in error:
org.keycloak.quickstart.springboot.ProductServiceTest.org.keycloak.quickstart.springboot.ProductServiceTest
  Run 1: ProductServiceTest.setup:32 » BadRequest HTTP 400 Bad Request
  Run 2: ProductServiceTest.cleanUp:60 » NotFound HTTP 404 Not Found
```

Reply:
I've run the tests for each quickstart related to this PR separately and everything passed. How did you run the tests?

Reply:
It's running fine for me. @abstractj Stupid question but was your KC server running ?

Reply:
@sebastienblanc yes! Let's just wait for the output from Travis. I believe you and Travis are more reliable than me when it comes to Spring Boot.